nixos/tests/keter.nix

```diff
@@ -0,0 +1,47 @@
+import ./make-test-python.nix ({ pkgs, ... }:
+  let
+    port = 81;
+  in
+  {
+    name = "keter";
+    meta = with pkgs.lib.maintainers; {
+      maintainers = [ jappie ];
+    };
+
+
+    nodes.machine = { config, pkgs, ... }: {
+      services.keter = {
+        enable = true;
+        keterPackage = pkgs.haskellPackages.keter;
+
+        globalKeterConfig = {
+          listeners = [{
+            host = "*4";
+            inherit port;
+          }];
+        };
+        bundle = {
+          appName = "test-bundle";
+          domain = "localhost";
+          executable =
+            let
+              exe = pkgs.writeShellScript "run" ''
+                ${pkgs.python3}/bin/python -m http.server $PORT
+              '';
+            in
+            "${exe}";
```

Comment:
Is there really a reason you have to wrap this? What breaks?

(Also, this could be a one-liner: `executable = "${pkgs.writeShellScript "run" "${pkgs.python3}/bin/blahblahyoucanfillintherest"}"`

Reply:
there was indeed no need to wrap this.
I didn't make it a 1 liner because that would make it go over 80 char limit.